docs/en/stack/ml/dataframeanalytics.asciidoc

```diff
@@ -0,0 +1,98 @@
+[role="xpack"]
+[[ml-dfanalytics]]
+=== {dfanalytics-cap}
+
+{dfanalytics-cap} allows you to perform different analyses of your data and 
+annotate it with the results. Essentially, as part of its output, {dfanalytics} 
+appends the results of the analysis to the source data. By doing this, it 
+provides additional insights on the data. The process leaves the source index 
+intact, it creates a new index that contains the annotated data. You can slice 
```

Comment:
`... it creates a new index that contains a copy of the source data and the annotated data.`